### PR TITLE
Allow plank and branch bundles to be made from cordage, vines, or wire

### PR DESCRIPTION
**Pull request by wapcaplet**
#### Summary

SUMMARY: Content "Allow plank and branch bundles to be made from cordage, vines, or wire"

#### Purpose of change

Provide more options for wood-hauling, particularly for innawoods characters.

Contributes to #41182 
Enhances #41573 

#### Describe the solution

Allow the existing "bundle of planks" and "bundle of stout branches" to be made from equivalent wire, vine, or cordage ropes, in addition to regular 6-foot ropes. Also add a "bundle of long stout branches", requiring two pieces of wire, vine, rope, or cordage.

Vague up the bundle descriptions not to specifically mention rope.

#### Describe alternatives you've considered

Considered allowing various long ropes too, but the idea is to make neat bundles without a bunch of excess rope. Disassemble your long ropes if you need short ropes to make bundles.

#### Testing

Spawn materials for constructing bundles:
- rope, wire, vine, cordage rope
- plank, stout branch, long stout branch

Construct bundles from different materials, and:
- check that item descriptions show correct components
- check disassembly menu `(` to be sure correct components are indicated
- disassemble bundles and make sure original materials are yielded

#### Additional context

New component options in recipe menu:

image

New bundle of 5 long stout branches made from short cordage rope:

image

Thanks @chaosvolt for suggesting this easy improvement.

### CODE REVIEW

**Comment by I-am-Erk** [comment]
I believe there is a requirement group for cordage you could use.

File: data/json/requirements/materials.json [hunk 1]
```diff
@@ -119,6 +119,12 @@
     "//": "Materials used for lashing, when choice of makeshift rope is sensible, per 216 g/180 cm of rope.",
     "components": [ [ [ "rope_6", 1 ], [ "vine_6", 1 ], [ "rope_makeshift_6", 1 ] ] ]
   },
+  {
+    "id": "rope_any_short",
+    "type": "requirement",
+    "//": "Materials used for lashing, when any rope or wire is sensible, per 216 g/180 cm of rope.",
+    "components": [ [ [ "rope_6", 1 ], [ "wire", 1 ], [ "vine_6", 1 ], [ "rope_makeshift_6", 1 ] ] ]
```
Comment (I-am-Erk) [inline data/json/requirements/materials.json:126]:
```suggestion
    "components": [ [ [ "rope_natural_short", 1 ], [ "wire", 1 ] ] ]

**Comment by I-am-Erk** [comment]
Great! We could use someone to check out where we're using rope_natural_short and see if rope_any_short would fit, but that's definitely outside this PR.

**Comment by I-am-Erk** [comment]
Dang I didn't notice you still had a WIP tag on this, do you want me to revert it?

**Comment by wapcaplet** [comment]
> Dang I didn't notice you still had a WIP tag on this, do you want me to revert it?

@I-am-Erk No, it was close enough, and functional as-is. We talked about adding copper wire or string to the new requirement but it can wait for a future PR. Thanks!

**Pull request merged**